app/access.py

```diff
@@ -40,7 +40,8 @@ def check_acls(method):
         @auth_or_token
         @functools.wraps(method)
         def wrapper(self, *args, **kwargs):
-            if not set(acl_list).issubset(self.current_user.permissions):
+            if not (set(acl_list).issubset(self.current_user.permissions)
+                    or "System admin" in self.current_user.permissions):
```

Comment:
Looks like this needs a rebase/merge

Reply:
Frustrating--I did a rebase. What am I doing wrong?